manifests/3/3DVista/VirtualTour/25.0.3.0/3DVista.VirtualTour.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: 3DVista.VirtualTour
+PackageVersion: 2025.0.3.0

```

Comment:
@rovabunw - The folder the manifests are in must match the `PackageVersion` in the manifest.